src/main/java/by/glinkiy/lesson1/task3/Time.java

```diff
@@ -0,0 +1,64 @@
+package by.glinkiy.lesson1.task3;
+/**
+ * Имеется 4500 секунд. Вывести в консоль содержащихся в этом количестве секунд:
+ * А) минут + секунд,
+ * В) часов + минут + секунд,
+ * С) дней + часов + минут + секунд,
+ * D) недель + дней + часов + минут + секунд.
+ * по аналогии с примером выше.
+ */
+public class Time {
+    public static void main(String[] args) {
+        System.out.println(minAndSecond(4500));
+        System.out.println(hourMinSec(4500));
+        System.out.println(day(4500));
+        System.out.println(weeks(4500));
+
+
+    }
+
+    public static String minAndSecond(int value) {
+        int second = value;
```

Comment:
you don't need this variable
your method should looks like
public static String hourMinSec(int second) {